lib/rubydoop/job_runner.rb

```diff
@@ -0,0 +1,22 @@
+# encoding: utf-8
+
+require 'rubydoop/dsl'
+
+module Rubydoop
+  class JobRunner < Java::OrgApacheHadoopConf::Configured
+    include Java::OrgApacheHadoopUtil::Tool
+    def run(args)
+      job_setup_script, *rest = args
+      conf = Java::OrgApacheHadoopMapred::JobConf.new(get_conf, get_class)
+      conf.set(Java::Rubydoop::InstanceContainer::JOB_SETUP_SCRIPT_KEY, job_setup_script)
+      $rubydoop_context = Context.new(conf, rest)
+      begin
+        require job_setup_script
+      rescue => e
+        raise JobRunnerError, sprintf('Could not load job setup script (%s): %s', job_setup_script.inspect, e.message.inspect)
```

Comment:
You could add `, e.backtrace` to the end of this line and it would make debugging easier (I've wished this error had the right backtrace a few times).